Rust/src/rcf.rs

```diff
@@ -31,6 +37,20 @@ pub(crate) fn damp(x: usize, y: usize) -> f64 {
 
 const max_number_in_summary : usize = 5;
 
+fn add_to(a: &f64, b : &mut f64){
```

Comment:
Why do we need these specialized functions? Can't we do the dereferencing inline?

Reply:
For this case we actually can use Sum() -- but that would not fit the pattern of {a1,a2,a3....} being collected in b (which is not necessarily the same type as that of a1, a2 ...). Maybe there is a better way to rewrite the single visitor traversal we are using now (for example when type of a is the same as that of b).

Reply:
moved to util. A subsequent PR may clean up the traversal.